client/components/catalog/TagFilter.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <v-menu :close-on-content-click="false" offset-y>
+    <template v-slot:activator="{ on }">
+      <v-btn v-on="on" icon text>
+        <v-icon color="primary lighten-4">mdi-tag-outline</v-icon>
+      </v-btn>
+    </template>
+    <v-list>
+      <v-list-item
+        v-for="it in items"
+        :key="it.id"
+        @click="toggleTagFilter(it)">
+        <v-list-item-action class="mr-2">
+          <v-checkbox :value="it.isSelected" />
+        </v-list-item-action>
+        <v-list-item-content class="text-left">
+          <v-list-item-title>{{ it.name }}</v-list-item-title>
+        </v-list-item-content>
+      </v-list-item>
+    </v-list>
+  </v-menu>
+</template>
+
+<script>
+import find from 'lodash/find';
+import map from 'lodash/map';
+import { mapState } from 'vuex';
+
+export default {
+  computed: {
+    ...mapState('repositories', ['tags', 'tagFilter']),
+    items() {
+      return map(this.tags, it => {
+        const isSelected = find(this.tagFilter, item => item.id === it.id);
```

Comment:
You can use shorthand `find(this.tagFilter,  { id: it.id })`. Also make sure you cast it to boolean by putting !! before (otherwise isSelected might be an object)